services-api/src/main/java/io/scalecube/services/api/ServiceMessage.java

```diff
@@ -171,7 +173,7 @@ public boolean isError() {
    * @return error type.
    */
   public int errorType() {
-    String errorType = headers.get(ERROR_TYPE);
+    String errorType = headers.get(HEADER_ERROR_TYPE);
     if (errorType == null) {
       return -1;
```

Comment:
Plz move `-1` to constant called `NULL_ERROR_TYPE`, and udpated javadoc on `errorType()` method accordingly.